src/google/protobuf/compiler/main.cc

```diff
@@ -68,5 +69,10 @@ int main(int argc, char* argv[]) {
   cli.RegisterGenerator("--ruby_out", &rb_generator,
                         "Generate Ruby source file.");
 
+  // CSharp
+  google::protobuf::compiler::csharp::Generator csharp_generator;
+  cli.RegisterGenerator("--csharp_out", &csharp_generator,
```

Comment:
I think csharp_out makes sense - it's probably clearer to users who *don't* want C#, and no less clear to those who do... and it's consistent with ruby_out rather than rb_out.